src/es/dragontranslationnet/src/eu/kanade/tachiyomi/extension/es/dragontranslationnet/DragonTranslationNet.kt

```diff
@@ -1,83 +1,117 @@
 package eu.kanade.tachiyomi.extension.es.dragontranslationnet
 
-import eu.kanade.tachiyomi.multisrc.madara.Madara
 import eu.kanade.tachiyomi.network.GET
 import eu.kanade.tachiyomi.source.model.FilterList
 import eu.kanade.tachiyomi.source.model.MangasPage
 import eu.kanade.tachiyomi.source.model.Page
 import eu.kanade.tachiyomi.source.model.SChapter
 import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
 import eu.kanade.tachiyomi.util.asJsoup
 import okhttp3.HttpUrl.Companion.toHttpUrl
 import okhttp3.Request
 import okhttp3.Response
 import org.jsoup.nodes.Document
 import org.jsoup.nodes.Element
 
-class DragonTranslationNet : Madara("DragonTranslation.net", "https://dragontranslation.net", "es") {
+class DragonTranslationNet : ParsedHttpSource() {
 
-    override fun popularMangaRequest(page: Int): Request {
-        return GET("$baseUrl/mangas?page=$page", headers)
-    }
+    override val name = "DragonTranslation.net"
+
+    override val baseUrl = "https://dragontranslation.net"
+
+    override val lang = "es"
 
-    override fun popularMangaSelector() = "div:has(> div.series-card)"
+    override val supportsLatest = true
+
+    // Popular
+
+    override fun popularMangaSelector() = "article.card"
 
     override fun popularMangaNextPageSelector() = "li.page-item a[rel=next]"
 
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl/mangas?page=$page", headers)
+    }
+
     override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        setUrlWithoutDomain(element.select("div.series-box a").attr("href"))
-        thumbnail_url = element.select("img.thumb-img").attr("abs:src")
-        title = element.select(".series-title").text()
+        setUrlWithoutDomain(element.selectFirst("a.lanzador")!!.attr("href"))
+        thumbnail_url = element.selectFirst("img")?.attr("abs:src")
+        title = element.selectFirst("h2")!!.text()
     }
 
+    // Latest
+
+    override fun latestUpdatesSelector(): String = throw UnsupportedOperationException()
+
+    override fun latestUpdatesNextPageSelector(): String = throw UnsupportedOperationException()
```

Comment:
consider using `HttpSource` directly since you aren't really using parsed methods that much